os/fs/smartfs/README.txt

```diff
@@ -1,17 +1,20 @@
-SMARTFS README
-^^^^^^^^^^^^^^
+jSmartFs README
+^^^^^^^^^^^^^^^
 
 This README file contains information about the implemenation of the TinyAra
-Sector Mapped Allocation for Really Tiny (SMART) FLASH file system, SMARTFS.
+Journaling - Sector Mapped Allocation for Really Tiny FLASH file system, jSmartFs.
+Journaling tries to provide an assurance of file system integrity during system crash.
+By writing changes to a journal quickly, we can ensure that all changes to files
+are recorded and not lost during power shutdowns or crashes.
```

Comment:
Our journaling mechanism also provides filesystem consistency even when the garbage collector moves the sectors that have valid data. We need to explain this point as well in the readme file.